### PR TITLE
Mr/nullable

### PR DESCRIPTION
**Pull request by majodev**
In context to https://github.com/allaboutapps/nullable/pull/2 - make it easier for us to catch generation problems earlier with nullable after upgrades to goswagger.

No sure if we should add dedicated binding testing from this to go-starter, feels like this should continue to live in allaboutapps/nullable.

### CODE REVIEW

**Comment by majodev** [comment]
btw: we will need to switch to allaboutapps/nullable again before merging this of course. https://github.com/allaboutapps/nullable/pull/2 comes first.

**Review by mwieser — APPROVED** [review]
Nice, smart idea.

**Review by MorpheusXAUT — APPROVED** [review]
I'd say the actual bind testing (as added [here]( https://github.com/allaboutapps/nullable/pull/2/files#diff-26a8a16a47cd52bf52e0cdbdfd3f54195240d0c5415dd156dfe9bd33f4ec290a)) should be good enough and can continue to live in the `nullable` package itself - copying it to here seems a bit redundant, especially since you might potentially not even use any of the `nullable` types.

LGTM (given the package-change once the other PR is merged), thank you!

**Pull request merged**
